logs/repl/README.md

```diff
@@ -6,4 +6,4 @@ This directory is used by default to store the REPL sockets to all of showdown's
 
 The intended uses of these REPL sockets are for debugging (especially when the server is seemingly frozen) and scripting.
 
-You can use any tool capable of talking to Unix sockets such as `nc`. e.g., `nc -U app`
\ No newline at end of file
+You can use any tool capable of talking to Unix sockets such as `nc`. e.g., `nc -U app`
```

Comment:
I have no idea what you changed here, but you might as well remove the comma after `e.g.` if you're fixing grammar.